.github/actions/run-interop-ping-test/action.yml

```diff
@@ -0,0 +1,78 @@
+name: 'libp2p ping interop test'
+description: 'Run the libp2p ping interoperability test suite'
+inputs:
+  test-filter:
+    description: "Filter which tests to run out of the created matrix"
+    required: false
+    default: ""
+  extra-versions:
+    description: "Space-separated paths to JSON files describing additional images"
+    required: false
+    default: ""
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        WORK_DIR=$(realpath "${{ github.action_path }}/../../../multidim-interop")
+        echo "WORK_DIR=$WORK_DIR" >> $GITHUB_OUTPUT
+      shell: bash
+      id: find-workdir
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 17
+
+    - name: Expose GitHub Runtime # Needed for docker buildx to cache properly (See https://docs.docker.com/build/cache/backends/gha/#authentication)
+      uses: crazy-max/ghaction-github-runtime@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Install deps
+      working-directory: ${{ steps.find-workdir.outputs.WORK_DIR }}
+      run: npm ci
+      shell: bash
+
+    - name: Build images
+      working-directory: ${{ steps.find-workdir.outputs.WORK_DIR }}
+      run: make
+      shell: bash
+
+    - name: Run the test
+#      timeout-minutes: 30
```

Comment:
Uncomment or delete?

Reply:
I've left it for now to remind myself that it is a missing feature now. Is deleting fine?